core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -21,9 +21,6 @@
 import java.util.function.Consumer;
 import java.util.function.Function;
 
-import static com.oath.oak.Chunk.*;
```

Comment:
Wasn't/isn't it needed?

Reply:
The checkstyle forbids static imports. It makes less readable code.
So now each access to Chunk's static member would have to be something like `Chunk.someStaticMember` instead of `someStaticMember`.
I personally agree that this is clearer to the reader.